checkpoint/cp-resource-management/templates/_helpers.tpl

```diff
@@ -25,4 +25,47 @@ If release name contains chart name it will be used as a full name.
 {{- else -}}
     {{ default "default" .Values.serviceAccount.name }}
 {{- end -}}
+{{- end -}}
+
+{{/*
+  Return Dome9 subdomain in format xxN e.g. "eu1", "ap3" etc. For us* only return an empty string
+*/}}
+{{- define "dome9.subdomain" -}}
+{{- $datacenter := lower $.Values.datacenter -}}
+{{- if has $datacenter (list "us" "us1" "usea1") -}}
+{{- printf "" -}}
+{{- else if has $datacenter (list "eu" "eu1" "euwe1") -}}
+{{- printf "eu1" -}}
+{{- else if has $datacenter (list "ap1" "apse1") -}}
+{{- printf "ap1" -}}
+{{- else if has $datacenter (list "ap2" "apse2") -}}
+{{- printf "ap2" -}}
+{{- else if has $datacenter (list "ap" "ap3" "apso1") -}}
+{{- printf "ap3" -}}
+{{- else if has $datacenter (list "ap4" "apne1") -}}
+{{- printf "ap4" -}}
+{{- else if has $datacenter (list "ap5" "apea1") -}}
+{{- printf "ap5" -}}
+{{- else if has $datacenter (list "ca" "ca1" "cace1") -}}
+{{- printf "ca1" -}}
+{{- else -}}
+{{- $err := printf "\n\nERROR: Invalid datacenter: %s (should be one of: 'usea1' [default], 'euwe1', 'apse1', 'apse2', 'apso1', 'apne1', 'apea1', 'cace1')"  $.Values.datacenter -}}

```

Comment:
are we sure this is the correct set of regions for now?

Reply:
when we discussed it in the past with you and Yonatan, we agreed it's to include also regions that were not published yet. then the list is correct.